src/hazelweb/gui/OptionsPanel.re

```diff
@@ -165,6 +165,37 @@ let view =
             ~on_change=() => inject(ToggleMemoizeDoc),
             model.memoize_doc,
           ),
+          // Node.button(
```

Comment:
you should put this button down at the bottom next to the "Serialize to console" button